### PR TITLE
update minor misspelling of the script name

### PR DESCRIPTION
**Pull request by contractorwolf**
added some additional comments on where to obtain the script parameters

*Issue #, if available:*
the linked script had a different actual name than the codeblock showed

*Description of changes:*
my change edits the code block to reflect the correct script name.  The other comments were just about clarifying where the parameters can be obtained, since the examples had just partial replacements and might be confusing to some (i.e. they might just lookup their api__id and not replace the others).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mauerbac** [comment]
Thanks! Let me have the @aws-amplify/amplify-cli team review this!

**Comment by contractorwolf** [comment]
no problem @mauerbac, glad to help. you know anyone on the AppSync team? I have an question on one issue and another issue that I am dealing with that doesnt seem to have anyone ever taking a look at:
my question:
https://github.com/aws/aws-appsync-community/issues/144

an issue I am seeing as well:
https://github.com/aws/aws-appsync-community/issues/125

**Review by dabit3 — APPROVED** [review]

**Comment by dabit3** [comment]
> no problem @mauerbac, glad to help. you know anyone on the AppSync team? I have an question on one issue and another issue that I am dealing with that doesnt seem to have anyone ever taking a look at:
> my question:
> [aws/aws-appsync-community#144](https://github.com/aws/aws-appsync-community/issues/144)
> 
> an issue I am seeing as well:
> [aws/aws-appsync-community#125](https://github.com/aws/aws-appsync-community/issues/125)

cc @awsed

**Comment by dabit3** [comment]
Thanks for the PR @contractorwolf, LGTM. Waiting on 1 more approval.

File: docs/cli/graphql-transformer/searchable.md
Comment (jamesonwilliams) [inline docs/cli/graphql-transformer/searchable.md:145]:
```suggestion
  --esarn 'arn:aws:dynamodb:us-west-2:123456789xxx:table/Post-<api__id>-dev/stream/2019-20-03T00:00:00.350' # Event source ARN, copy the full DynamoDB table ARN
```

Comment (contractorwolf) [reply docs/cli/graphql-transformer/searchable.md:145]: not sure if that accept your suggested change, I dont usually use the web based way of making PR's because I usually have the project on my local.  Let me know if that works, sorry about that, crossed wires, arduinos on the brain

**Review by jamesonwilliams — APPROVED** [review]

**Review by dabit3 — APPROVED** [review]

**Comment by contractorwolf** [comment]
@dabit3 are we still waiting on aws-amplify/amplify-cli? isnt that your team? I cant see that group or I might try to nudge them

**Comment by attilah** [comment]
@contractorwolf thanks for the PR, this is a typo, referenced file can be found here: https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-elasticsearch-transformer/scripts/ddb_to_es.py

**Review by attilah — APPROVED** [review]
LGTM

**Pull request merged**
